docs/source/examples/external_sp2_output.txt

```diff
@@ -1,10 +1,10 @@
 
 Cost
 ----
- 0.7854
+ 0.7071
 
 Free Variables
 --------------
 x : 0.7854
-y : 0.7854
+y : 0.7071
```

Comment:
oh my bad, I was looking at an earlier example. It should be sin(pi/4). Agreed there's an issue on master.